### PR TITLE
man: cleanup and update manual page

### PR DESCRIPTION
**Pull request by jonathangray**
Remove roff escapes, sort and add missing options.

### CODE REVIEW

**Comment by dwatteau** [comment]
Thanks for this, it was sitting somewhere on my todo-list for a while.

I'm reviewing this, but indeed some documented options are outdated, when comparing them with the code. I'll add some further suggestions to this PR.

File: dists/scummvm.6 [hunk 1]
```diff
@@ -14,226 +13,400 @@
...
+.Ar amiga ,
+.Ar atari ,
+.Ar c64 ,
+.Ar fmtowns ,
+.Ar mac ,
+.Ar nes ,
+.Ar pc ,
+.Ar pce ,
+.Ar segacd ,
+.Ar windows
```
Comment (sev-) [inline dists/scummvm.6:336]: We have more platforms. The full list is at https://github.com/scummvm/scummvm/blob/master/common/platform.cpp#L28

Comment (jonathangray) [reply dists/scummvm.6]: added a commit with the rest of the first strings in the table

File: dists/scummvm.6 [hunk 2]
```diff
@@ -14,226 +13,423 @@
...
+.Ar ega ,
+.Ar vga ,
+.Ar amiga ,
+.Ar fmtowns ,
+.Ar pc9821 ,
+.Ar pc9801 ,
+.Ar 2gs ,
+.Ar atari ,
+.Ar macintosh
+.Pc .
```
Comment (dwatteau) [inline dists/scummvm.6:382]:
```suggestion
.Po
.Ar hercGreen ,
.Ar hercAmber ,
.Ar cga ,
.Ar cgaComp ,
.Ar cgaBW ,
.Ar ega ,
.Ar vga ,
.Ar amiga ,
.Ar fmtowns ,
.Ar pc9821 ,
.Ar pc9801 ,
.Ar 2gs ,
.Ar atari ,
.Ar macintosh ,
.Ar macintoshbw
.Pc .
```

from:

https://github.com/scummvm/scummvm/blob/1c7e799f510c6734eaaecda7ee76d3f0963725c1/common/gui_options.cpp#L63-L77

(commandLine.cpp and doc/ seem out-of-date too… will provide a separate PR for that)

File: dists/scummvm.6 [hunk 3]
```diff
@@ -14,226 +13,423 @@
...
+or all targets if none is specified.
+.Fl -game= Ns Ar ID
+In combination with
+.Fl -add
+or
+.Fl -detect
+only adds or attempts to detect the game with id
+.Ar ID .
+.It Fl -list-themes
+Display list of all usable GUI themes.
```
Comment (dwatteau) [inline dists/scummvm.6:279]:
```suggestion
.It Fl -list-themes
Display list of all usable GUI themes and exit.
```

this one exits, too

**Review by dwatteau — COMMENTED** [review]
Thank you very much for this, I've made some suggestions, if you're OK with them.

(I have some other suggestions, but I'll submit my own PR after this one, since they're not related to the lines you are changing.)

File: dists/scummvm.6 [hunk 4]
```diff
@@ -14,226 +13,423 @@
...
+.It Ar windows
+Windows built in MIDI sequencer for Windows users.
+.El
+.It Fl F | -no-fullscreen
 Force windowed mode.
-.It Fl g, -gfx-mode= Ns Ar MODE
+.It Fl f | -fullscreen
+Force full-screen mode.
+.It Fl g | -gfx-mode= Ns Ar MODE
 Select graphics scaler:
```
[the remark this answers is not in the document]
Comment (jonathangray) [reply dists/scummvm.6]: Unfortunate that -g hq3x and the like no longer work and two long options are required instead.

**Comment by jonathangray** [comment]
thanks, all of the suggestions are now in the branch

**Comment by dwatteau** [comment]
Thank you very much! LGTM, let's wait a bit in case someone sees something else.

Once it's merged, I'll open my own PR with some further changes, but this first batch looks very good as is.

**Comment by digitall** [comment]
@jonathangray : Thanks for the improvements to the ScummVM Manual page.

One minor point, could you rebase and edit these commits to change the prefix from "man: ..." to "DOC: MAN: ..." or similar as per https://wiki.scummvm.org/index.php?title=Commit_Guidelines#Commit_message_formatting

If you don't know how to do this or don't feel able to, I or another of the team should be able to do this to the PR (since team members have permissions to edit the PR branches).

**Comment by jonathangray** [comment]
changed prefix to DOCS: MAN: and rebased

**Comment by digitall** [comment]
@jonathangray : Thanks for sorting that. Previous commits to the manual have been prefixed by "DOC:", but DOCS should be fine as well.

**Comment by sev-** [comment]
Thank you, merging.

**Pull request merged**